app/src/main/java/edu/agh/susgame/front/gui/components/menu/components/createlobby/CreateLobbyView.kt

```diff
@@ -41,58 +44,75 @@ fun CreateLobbyView(
     var gameTime by remember { mutableIntStateOf(DEFAULT_GAME_TIME) }
     var isGamePinEnabled by remember { mutableStateOf(false) }
 
-    Column(Modifier.padding(PaddingL)) {
-        Column(
-            Modifier
-                .padding(PaddingL)
-                .weight(1f)
-                .verticalScroll(rememberScrollState())
+    MenuBackground()
+
+    Column(
+        Modifier
+            .fillMaxSize()
+            .padding(PaddingL),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Spacer(modifier = Modifier.weight(0.1f))
+        Header(
+            title = Translation.Menu.CREATE_GAME, style = TextStyler.TerminalXL
+        )
+        Spacer(modifier = Modifier.weight(0.1f))
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .weight(3.5f)
         ) {
 
-            Header(
-                title = Translation.Menu.CREATE_GAME
-            )
+            Column(
+                Modifier
+                    .padding(PaddingL)
+                    .fillMaxWidth(0.5f)
+                    .verticalScroll(rememberScrollState()),
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
 
-            GameNameComp(
-                gameName = gameName,
-                onGameNameChange = { gameName = it }
-            )
+                GameNameComp(
+                    gameName = gameName,
+                    onGameNameChange = { gameName = it }
+                )
 
-            GamePinComp(
-                gamePin = gamePin,
-                showPassword = showPassword,
-                isGamePinEnabled = isGamePinEnabled,
-                onGamePinChange = { gamePin = it },
-                onShowPasswordChange = { showPassword = it },
-                onGamePinEnabledChange = { isGamePinEnabled = it }
-            )
+//                GamePinComp(
+//                    gamePin = gamePin,
+//                    showPassword = showPassword,
+//                    isGamePinEnabled = isGamePinEnabled,
+//                    onGamePinChange = { gamePin = it },
+//                    onShowPasswordChange = { showPassword = it },
+//                    onGamePinEnabledChange = { isGamePinEnabled = it }
+//                )
```

Comment:
Tutaj dodaj może jakiś komentarz z `TODO` typu:

```kotlin
// TODO Handle game PIN on backend, or remove this code
```

Albo w ogóle możesz usunąć te linie

Reply:
wyszło na to że chcemy to przywrócić, dodam ticket do zaimplementowania tego na backu i froncie